pages/userSettings/userSettings.ejs

```diff
@@ -13,6 +13,59 @@
     <div id="content" class="container">
       <h1 class="mb-4">Settings</h1>
 
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex align-items-center">
+          Logged-in user profile information
+          <a href="/pl/logout" class="btn btn-danger btn-sm ml-auto">Logout</a>
+        </div>
+          <table class="table table-sm two-column-description">
+            <tbody>
+              <tr>
+                <th>User ID (UID)</th>
```

Comment:
"User ID" actually means something different to us. I'd be inclined to just call this `UID` - that has a well-known meaning to most people who would need to interpret this information.

```suggestion
                <th>UID</th>
```

Reply:
I would like to put something there in addition to a techy acronym. What words would you use to describe UID and UIN?

Reply:
We already use the term "UID" throughout PL (think access rules), so I don't think it'll be confusing to folks. If you want to give UIN a human-readable name, I might go with "unique identifier"?